pkg/cmd/pulumi/convert/io.go

```diff
@@ -33,11 +35,28 @@ func LoadConverterPlugin(
 	name string,
 	log func(sev diag.Severity, msg string),
 ) (plugin.Converter, error) {
+	var version *semver.Version
```

Comment:
We should be able to reuse `NewPluginSpec` here: https://github.com/pulumi/pulumi/blob/22cb0ad4da73c59a0ad4ccc220c3869a9ff966ff/sdk/go/common/workspace/plugins.go#L867

Reply:
Thanks! That is definitely the way I should have done it!